CSS/Components/alert.css

```diff
@@ -0,0 +1,45 @@
+.alert {
+  width: 100%;
+  padding: 0.8rem 1.5rem;
+  color: #fff;
+  background-color: var(--primary-color);
+  font-size: 1rem;
+  font-weight: 500;
+  box-shadow: 0 0 3px rgba(0, 0, 0, 0.5);
+  margin: 0.7rem 0;
+  border-radius: 4px;
+}
+
+.primary {
+  background-color: var(--primary-color);
+}
+.error {
+  background-color: var(--danger-color);
+}
+.success {
+  background-color: var(--success-color);
+}
+.warning {
+  background-color: var(--warning-color);
+}
```

Comment:
If you are not using there colors many times then don't create variables for it. Everything else looks good.

Reply:
Planning to use it other components as well, so added there. Thanks Prajwal